cg/cli/upload/delivery_report.py

```diff
@@ -210,30 +214,31 @@ def delivery_report(
     is_flag=True,
     help="run command without uploading to scout",
 )
-@click.pass_context
-def delivery_report_to_scout(context, case_id: str, dry_run: bool):
+@click.pass_obj
+def delivery_report_to_scout(context: CGConfig, case_id: str, dry_run: bool):
     """Fetches an delivery-report from housekeeper and uploads it to scout"""
-    if not context.obj.get("analysis_api"):
-        context.obj["analysis_api"] = MipDNAAnalysisAPI(context.obj)
-    analysis_api = context.obj["analysis_api"]
+    status_db: Store = context.status_db
+    housekeeper_api: HousekeeperAPI = context.housekeeper_api
+    scout_api: ScoutAPI = context.scout_api
 
     if not case_id:
-        suggest_cases_delivery_report(context, pipeline=Pipeline.MIP_DNA)
-        context.abort()
-
-    uploaded_delivery_report_files = analysis_api.housekeeper_api.get_files(
-        bundle=case_id,
-        tags=["delivery-report"],
-        version=analysis_api.housekeeper_api.last_version(case_id).id,
-    )
-    if uploaded_delivery_report_files.count() == 0:
+        suggest_cases_delivery_report(status_db=status_db, pipeline=Pipeline.MIP_DNA)
+        raise click.Abort
+
+    uploaded_delivery_report_files: List[hk_models.File] = [
```

Comment:
same as previously with multiqc, why not grab with first()?

Reply:
I answered above